storage/buckets/buckets_test.go

```diff
@@ -86,6 +86,32 @@ func TestCreateBucketClassLocation(t *testing.T) {
 	}
 }
 
+func TestCreateBucketDualRegion(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	bucketName := testutil.UniqueBucketName(testPrefix)
+	ctx := context.Background()
+
+	defer testutil.DeleteBucketIfExists(ctx, client, bucketName)
+
+	region1 := "US-WEST1"
+	region2 := "US-CENTRAL1"
+	if err := createBucketDualRegion(ioutil.Discard, tc.ProjectID, bucketName, region1, region2); err != nil {
+		t.Fatalf("createBucketClassLocation: %v", err)
+	}
+
+	attrs, err := client.Bucket(bucketName).Attrs(ctx)
+	if err != nil {
+		t.Fatalf("Bucket(%q).Attrs: %v", bucketName, err)
+	}
+
+	if !strings.Contains(attrs.Location, region1) || !strings.Contains(attrs.Location, region2) {
```

Comment:
Nit: we should probably verify the exact string here.

Reply:
The string returned may be either "region1+region2" or "region2+region1"
I received confirmation that the canonical order is deterministic and alphabetical so we could change that here but I'm hesitant to verify against an exact string when the order may be reversed and it means the same thing.